src/cmds/scripts/pbs_habitat.in

```diff
@@ -238,14 +238,16 @@ upgrade_pbs_database() {
 		echo "Database version file: ${data_dir}/PG_VERSION not found, cannot continue"
 		return 1
 	fi
-
+	
+	sys_pgsql_ver=$(echo `${PGSQL_DIR}/bin/psql -V` | awk '{print $NF}' | cut -d '.' -f 1,2)
 	ver=`cat ${data_dir}/PG_VERSION`
-	result=`echo ${ver} | awk '{ if ($0 < 9.3) {print 1} else {print 0} }'`
+
+	result=1; [[ $ver -ge $sys_pgsql_ver ]] && result=0
```

Comment:
Just for the sake of consistency with the file, if you choose to change, break this into one instruction per line.